Gruntfile.js

```diff
@@ -0,0 +1,30 @@
+/*
+	Copyright 2020 OCAD University.
+
+	Licensed under the BSD-3-Clause license.
+*/
+
+"use strict";
+
+module.exports = function (grunt) {
+	grunt.config.init({
+		lintAll: {
+			sources: {
+				md: [ "*.md", "!.github/*.md"],
+				js: ["*.js"],
+				json: ["*.json"]
+			}
+		},
+		lintspaces: {
+			jsonindentation: {
+				options: {
+					indentation: "tabs"
```

Comment:
As far as I can see, despite the convention in here being for tabs, this file itself is actually indented with single spaces (at least in the github interface). Our existing coding standards https://wiki.fluidproject.org/display/fluid/Coding+and+Commit+Standards defer to Crockford's at https://www.crockford.com/code.html which list spaces so unless there is a compelling reason might we return to the origin?
Terribly sorry for the bikeshedding!

Reply:
I think there is a legit movement towards use of tabs these days in non-Infusion projects. I was under the impression that We Count was one of the projects that had embraced this more accessibility-friendly (i.e. users who need very large fonts can choose smaller tab lengths for readability) convention?

Reply:
Yes, this is exactly the reason. The movement towards tabs in WeCount was initiated from a discussion of [this spaces vs tabs article](https://dev.to/alexandersandberg/why-we-should-default-to-tabs-instead-of-spaces-for-an-accessible-first-environment-101f), which we agreed it makes a lot sense.